scripts/jenkins/qa_openstack.sh

```diff
@@ -120,6 +120,9 @@ case "$cloudsource" in
     ;;
 esac
 
+# be sure to use packages from cloud repository
+$zypper dup --no-recommends --from cloud
```

Comment:
@dirkmueller https://github.com/SUSE-Cloud/automation/pull/508 removes ```cloud-init``` and it works. Thanks for the hint!